contribs/gnodev/cmd/gnodev/main.go

```diff
@@ -4,48 +4,55 @@ import (
 	"context"
 	"flag"
 	"fmt"
-	"io"
 	"log/slog"
-	"net"
 	"net/http"
 	"os"
 	"path/filepath"
-	"strings"
 	"time"
 
-	"github.com/charmbracelet/lipgloss"
-	"github.com/fsnotify/fsnotify"
-	"github.com/gnolang/gno/contribs/gnodev/pkg/dev"
 	gnodev "github.com/gnolang/gno/contribs/gnodev/pkg/dev"
 	"github.com/gnolang/gno/contribs/gnodev/pkg/emitter"
-	"github.com/gnolang/gno/contribs/gnodev/pkg/logger"
 	"github.com/gnolang/gno/contribs/gnodev/pkg/rawterm"
 	"github.com/gnolang/gno/contribs/gnodev/pkg/watcher"
-	"github.com/gnolang/gno/gno.land/pkg/gnoweb"
-	zaplog "github.com/gnolang/gno/gno.land/pkg/log"
+	"github.com/gnolang/gno/gno.land/pkg/integration"
 	"github.com/gnolang/gno/gnovm/pkg/gnoenv"
-	"github.com/gnolang/gno/gnovm/pkg/gnomod"
 	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/gnolang/gno/tm2/pkg/crypto"
+	"github.com/gnolang/gno/tm2/pkg/crypto/keys"
 	osm "github.com/gnolang/gno/tm2/pkg/os"
-	"github.com/muesli/termenv"
 )
 
 const (
 	NodeLogName        = "Node"
 	WebLogName         = "GnoWeb"
 	KeyPressLogName    = "KeyPress"
 	EventServerLogName = "Event"
+	AccountsLogName    = "Accounts"
+)
+
+var (
+	DefaultDeployerName    = integration.DefaultAccount_Name
+	DefaultDeployerAddress = crypto.MustAddressFromString(integration.DefaultAccount_Address)
+	DefaultDeployerSeed    = integration.DefaultAccount_Seed
 )
 
 type devCfg struct {
+	// Listeners
 	webListenerAddr          string
 	nodeRPCListenerAddr      string
 	nodeP2PListenerAddr      string
 	nodeProxyAppListenerAddr string
 
+	// Users default
+	deployKey          string
+	home               string
+	root               string
+	additionalAccounts varAccounts
```

Comment:
What do you think about renaming this to `premine`?

Reply:
was looking for some better name, thanks 🙏

Reply:
I've renamed it to `premineAccounts`. I've kept "account" at the end because it will be used by `-add-account`, which needs to define a single entity added (one by one if used multiple times).
Updated description:
```diff
@@ -126,9 +126,9 @@ func (c *devCfg) RegisterFlags(fs *flag.FlagSet) {
 	)
 
 	fs.Var(
-		&c.additionalAccounts,
+		&c.premineAccounts,
 		"add-account",
-		"add and set account(s) in the form `<bech32>[:<amount>]`, can be used multiple time",
+		"add (or set) a premine account in the form `<bech32|name>[:<amount>]`, can be used multiple time",
 	)
```

**Note:** I've kept the flag as it is, as I think `add-premine-account` would be a pain to write.